docs/docs/styles/grid-system.mdx

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 3
+---
+
+# Grid System
+
+The grid system used at We The Collective is loosely based on the foundation grid system. 
+
+Out of the box it provides a responsive flex grid system that is 1440px wide and contains 12 columns with a 24px gutter.
+
+
+## Before you start
+
+#### <a href="https://codepen.io/team/wtc/pen/BEjzmW" target="_blank" rel="noreferrer noopener">Bookmarklet</a>
+
+This bookmarklet will allow you to view implementations of this grid system in right in your browser.
+
+
+#### <a href="https://bootstrap.wethe.work/grid-demo/" target="_blank" rel="noreferrer noopener">Demo</a>
+
+This is a demo of the grid system. You can also access this file from the documentation project <a href="https://github.com/wethegit/bootstrap/blob/main/documentation/pages/grid-demo/index.pug" target="_blank" rel="noreferrer noopener">here</a>.
+
+## How to use
+
+The grid itself is available from medium up and there are classes provided that allow the configuration of elements to different grid sizes at different responsive stops.
+
+The grid system works around the concept of rows and columns. Each .row can contain .columns and each .column must contain both a .column class as well as a .column--X class.
+
+## Basic
+
+The simplest way to use the grid in practice is to create a row and, within that row, create some columns:
+
+```
+.row
+  .column.column--4
+    p I'm a 4 column element
+  .column.column--4
```

Comment:
Let's use the `scss` syntax highlighting for these code blocks, so that beginning code block tag can be:
```
```scss
```